app/controllers/flow/vehicle/owned_handler.rb

```diff
@@ -0,0 +1,18 @@
+module Flow
+  module Vehicle
+    class OwnedHandler
+      class << self
+        def model(session_data)
```

Comment:
Is there a benefit in DRYing up all these `.model` and `.form` methods? I'm asking as a genuine question.

Reply:
I'm not sure. I hope there isn't much duplication between them, but I have noticed that there's a (non-obvious) pattern around having to call map(&:to_s) when loading the form in from the session (as the session appears to not use HashWithIndifferentAccess)